src/App.css

```diff
@@ -2128,7 +2128,7 @@ code,
   position: absolute;
   height: 0.5em;
   /* margin to offset bullet width */
-  margin-left: -1em;
+  margin-left: 0.5em;
```

Comment:
Can you explain this value?

Reply:
It was to shift the DropEnd towards right, `1.5em` seemed just enough.